app/models/course/requirement.rb

```diff
@@ -0,0 +1,4 @@
+class Course::Requirement < ActiveRecord::Base
+  actable
+  belongs_to :has_requirement, polymorphic: true # naming is hard
```

Comment:
Sometimes it's very hard to give meanful names.. or how about dependable ?
Means this thing can depend on others.

On Saturday, February 28, 2015, Joel Low <notifications@github.com> wrote:

> In app/models/course/requirement.rb
> <https://github.com/Coursemology/coursemology2/pull/100#discussion_r25551682>
> :
>
> > @@ -0,0 +1,4 @@
> > +class Course::Requirement < ActiveRecord::Base
> > +  actable
> > +  belongs_to :has_requirement, polymorphic: true # naming is hard
>
> the "xxx-able" convention is good, but from experience it gives me
> completely meaningless names. what does 'requirable' mean? 'taggable' makes
> sense, but what kind of tags?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Coursemology/coursemology2/pull/100/files#r25551682>.
>